src/main/java/finalcountdown/homerecipesystembackend/controller/RecipeController.java

```diff
@@ -1,8 +1,32 @@
 package finalcountdown.homerecipesystembackend.controller;
 
-import org.springframework.stereotype.Controller;
+import finalcountdown.homerecipesystembackend.model.Recipe;
+import finalcountdown.homerecipesystembackend.service.RecipeService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
-@Controller
+import java.net.URI;
+
+@RestController
+@Slf4j
+@RequestMapping("/recipe")
 public class RecipeController {
+    private final RecipeService recipeService;
+
+    public RecipeController(RecipeService recipeService) {
+        this.recipeService = recipeService;
+    }
 
+    @PostMapping("/create")
+    public ResponseEntity<?> createRecipe(@RequestBody Recipe newRecipe) {
```

Comment:
We cannot save the Recipe database object, because Recipe has a PrepMethod type of prepMethod. 
So the @Requestbody should instead be RecipeRequest newRecipe. Check the same endpoint on IngredientController.

To save a Recipe to the database using JSON requestbody, we need to create a RecipeRequest DTO. 
RecipeRequest should contain these fields:
String name;
String note;
Long prepMethodId;

Check how IngredientRequest is done based on Ingredient class.